Tests/scripts/prepare_content_packs_for_testing.sh

```diff
@@ -3,28 +3,26 @@
 # exit on errors
 set -e
 
-CIRCLE_BRANCH=${CIRCLE_BRANCH:-unknown}
-CIRCLE_BUILD_NUM=${CIRCLE_BUILD_NUM:-00000}
-CIRCLE_ARTIFACTS=${CIRCLE_ARTIFACTS}
-PACK_ARTIFACTS=$CIRCLE_ARTIFACTS/content_packs.zip
-ID_SET=$CIRCLE_ARTIFACTS/id_set.json
+CI_COMMIT_BRANCH=${CI_COMMIT_BRANCH:-unknown}
+CI_BUILD_ID=${CI_BUILD_ID:-00000}
+ARTIFACTS_FOLDER=${ARTIFACTS_FOLDER}
+PACK_ARTIFACTS=$ARTIFACTS_FOLDER/content_packs.zip
+ID_SET=$ARTIFACTS_FOLDER/id_set.json
 EXTRACT_FOLDER=$(mktemp -d)
 
-if [[ -z "$GCS_MARKET_KEY" ]]; then
+if [[ ! -f "$GCS_MARKET_KEY" ]]; then
     echo "GCS_MARKET_KEY not set aborting!"
     exit 1
 fi
 
 echo "Preparing content packs for testing ..."
 
-KF=$(mktemp)
-echo "$GCS_MARKET_KEY" > "$KF"
-gcloud auth activate-service-account --key-file="$KF" > auth.out 2>&1
+gcloud auth activate-service-account --key-file="$GCS_MARKET_KEY" > auth.out 2>&1
 echo "Auth loaded successfully."
 
 # ====== BUILD CONFIGURATION ======
 GCS_BUILD_BUCKET="marketplace-ci-build"
-BUILD_BUCKET_PATH="content/builds/$CIRCLE_BRANCH/$CIRCLE_BUILD_NUM"
+BUILD_BUCKET_PATH="content/builds/$CI_COMMIT_BRANCH/$CI_BUILD_ID"
```

Comment:
Make sure we use the same path for `copy_and_upload_packs.py` in the Upload job, otherwise the upload will fail.